### DISCUSSION TITLE
MS Office, Pandoc, and Images

### DISCUSSION THREAD

**Discussion by jhomme** [jgm/pandoc#9665] [Q&A]
Hi,
Because I am a screen reader user, it's way easier for me to write in Markdown, and let Pandoc make documents. When I get a document in Word or PowerPoint and create a Markdown document from it and the document contains images, is there a way to get the images from the original document, so that I can point to them in Markdown using image syntax to put them back in after editing?

Thank you.

**Comment by jgm** [comment]
Yes.  See `--extract-media` in the manual.
`--extract-media=media` will put the media in the `media` directory and rewrite markdown links to point there.
